sim/core/spell_result.go

```diff
@@ -97,64 +97,57 @@ func (spell *Spell) PhysicalCritCheck(sim *Simulation, attackTable *AttackTable)
 	return sim.RandomFloat("Physical Crit Roll") < spell.PhysicalCritChance(attackTable)
 }
 
-// TODO: This should probably be merged with SpellDamage()? Doesn't make sense the way it is.
-func (spell *Spell) spellPower() float64 {
-	return spell.Unit.GetStat(stats.SpellPower) +
-		spell.BonusSpellPower +
-		spell.SpellSchoolPower() +
-		spell.Unit.PseudoStats.MobTypeSpellPower
-}
-
+// The combined bonus damage (aka spell power) for this spell's school(s).
 func (spell *Spell) SpellDamage() float64 {
-	return spell.spellPower() + spell.Unit.GetStat(stats.SpellDamage)
-}
+	var bonusDamage float64
 
-func (spell *Spell) SpellSchoolPower() float64 {
 	switch spell.SchoolIndex {
 	case stats.SchoolIndexNone:
 		return 0
 	case stats.SchoolIndexPhysical:
-		// Return correct value if ever used for a physical spell.
+		// PseudoStats.BonusDamage is physical "spell power", just return that here.
+		// TODO: Do "MobTypeSpellPower" effects for physical exist? E.g. something like "Do x extra weapon damage against y type"?
+		// If yes then it needs to be handled here and for the multi school case below.
```

Comment:
They are on the weapon, so I'd guess they would work like the other weapon damage increases on weapons, just only against beasts. That would be handled outside of any spell code anyway.

And I'm pretty sure that aura effect is simply physical only bonus damage against creature types, if used on normal auras.

Considering the wording on the old relic https://www.wowhead.com/classic/spell=2635/beast-slayer it was probably intended to be bonus damage for all schools, but was then changed to be physical only which its usage on https://www.wowhead.com/classic/item=209678/mantle-of-the-thresher-slayer implies. For magic bonus damage the effect MOD_FLAT_SPELL_DAMAGE_VERSUS was created, e.g. for https://www.wowhead.com/classic/item=23207/mark-of-the-champion

If that unimplemented BFD shoulder remains the only item it can probably be ignored for now.

Reply:
Definitely.